lib/src/bloc/i_movies_bloc.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:async';
+
+import '../model/movies_model.dart';
+import 'bloc.dart';
+
+abstract class IMoviesBloc extends Bloc {
+  Stream<MoviesModel> get stream;
+
+  void fetchMovies();
+}
```

Comment:
missed an extra line here, do Ctrl + Alt + L to fix it or do it manually

Reply:
fixed